### ISSUE TITLE
Status unavailable in Loveace card since update

### ISSUE THREAD

**Issue by puttemanp** [barleybobs/homeassistant-ecowater-softener#18]
Hello,
I updated to the latest version, but since the status of all entitities is unavailable in the card I created: https://imgur.com/a/bZXPiXK

However, when I folow the procedure in topic 'Only seeing one entity #13' I can correctly look up the water softener and click on attributes. All data is visible. 

Any insights?

**Comment by barleybobs** [comment]
It should only produce one entity. I am not sure how you have got all of those entities unless they are template sensors. 

If you could provide more details that would be great.

Thanks 👍

**Comment by puttemanp** [comment]
When I first installed Ecowater these binary sensors where created: https://imgur.com/a/8Wn1GjR
Did nothing special to obtain them.

**Comment by barleybobs** [comment]
The integration only creates one entity. Did you use to use [https://community.home-assistant.io/t/ecowater-softener-how-to-integrate-this-python-script/177263](https://community.home-assistant.io/t/ecowater-softener-how-to-integrate-this-python-script/177263), as it looks like they were created by this script.

Thanks 👍

**Comment by puttemanp** [comment]
I checked and indeed, my appdeamon folder contains the script and the configuration. I replaced www.wifi <http://www.wifi>  by wifi and restarted. As the script only runs once every 60 minutes I’ll have to wait an hour to see if data gets updated and if I get my values again.

 

Thx for pointing me in the right direction. I had completely forgotten I had installed both your integration and the appdeamon script.

 

P.
